.travis.yml

```diff
@@ -1,38 +1,101 @@
+conditions: v1
+
 language: python
-python:
-- "3.5"
-- "3.6-dev"
-- "3.7-dev"
-- "nightly"
 
 dist: trusty
 sudo: false
 cache: pip
 
 before_install:
 - pip install --upgrade flit
-install:
-- pushd blurb
-- flit install
-- popd
-script:
-- blurb test
 
 .mixtures:
+- &run-if-tagged
+  if: tag IS present
+- &run-if-cherry-picker
+  if: tag =~ ^cherry\-picker\-v\d+\.\d+\.\d+$
```

Comment:
@Mariatta I'm a bit concerned about restricting this to `vX.Y.Z` format, while in real life there might be some extensions (dev/a/b). Should I extend this regexp to take those into account?

Reply:
I think we can leave the regex as is (handling vX.Y.Z) and when the need arise we can adjust it later.